### ISSUE TITLE
Loosen unidiff pinning

### ISSUE THREAD

**Issue by jonashaag** [danvk/webdiff#236]
Right now there is a pin to 0.7.4. Is this on purpose or can be make it less strict, say `>=0.7.4,<0.8`?

**Comment by danvk** [comment]
Yes, it could be made less strict. `^0.7.4` is fine. I'd be happy to merge a PR for this.

I don't think there's any particular reason for this precise pin. It goes back to when I was using `setup.py` years ago: https://github.com/danvk/webdiff/pull/150/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R29
